src/Models/UpstreamStatus.php

```diff
@@ -55,6 +62,22 @@ public function getUpdates()
         return $this->updates;
     }
 
+    /**
+     * Retrives composer dependecy updates
+     *
+     * @return object
+     */
+    public function getComposerUpdates()
+    {
+        if ($this->composerUpdates === null) {
+            $env = $this->getEnvironment();
+            $this->composerUpdates = $this->request()->request(
+                "sites/{$env->getSite()->id}/environments/{$env->id}/build/updates"
+            )['data'];
```

Comment:
Is `$this->request()->request()` always guaranteed to return an assoc array? If not we should bring the `['data']` down to its own line and do a check before.

Reply:
Looking at the `request()` method code I see there always is a data key that's being returned, as per  https://github.com/pantheon-systems/terminus/blob/7cdc84884017a47692d42189ffe85a1ec5d903dc/src/Request/Request.php#L206-L222 . It seems to be the preferred pattern which is used across the repo.